James/book_challenge.py

```diff
@@ -0,0 +1,112 @@
+'''
+Object-oriented book challenge! 
+
+Your challenge for today is to complete the fill out the missing parts of the Booklist class in book_class.py
+Then, test each method for the class Booklist!
+
+Hint: all methods for the Booklist class go in the book_class.py script
+This class and the methods are already imported in this script -- test them here for specific cases!
+'''
+
+# import the Booklist class and corresponding methods
+from book_class import *
+
+
+'''
+PART #1. 
+Set up the initialization function for class Booklist()
+The function should initialize just 1 attribute, called 'books', as an empty list
+No parameters needed, other than self
+
+Then, create an object of class Booklist called 'my_library' in this script.
+Finally, print the books attribute of my_library -- it should be an empty list
+Also, print out the type of my_library to see what you get :) 
+'''
+print('PART 1\n')
+my_library = Booklist()
+print(my_library.books)
+print(type(my_library))
+print()
+
+'''
+Part #2: 
+Define the add() method to add a book to an object of class Booklist
+-The method should take 2 parameters other than self, both strings -- 'title', and 'author'
+-The method should make a book variable as a dictionary with two key/value pairs for title/author
+-Then, the method should append this book to books attribute of the the Booklist object  - i.e. self.books.append(book)
+
+Once you have finished the method, add the following books to your booklist:
+Just Mercy - Bryan Stevenson
+The New Jim Crow - Michelle Alexander
+The Truths We Hold - Kamala Harris
+My Grandmother's Hands - Resmaa Menakem
+
+Finally, prinb the books attribute of my_library to make sure your books have been added!
+'''
+
+print('\nPart 2\n')
+my_library.add( title ='Just Mercy', authors='Bryan Stevenson')
```

Comment:
This add function is correctly formatted here, but does not work because the add function is not correct